doc/admin-guide/plugins/traffic_dump.en.rst

```diff
@@ -0,0 +1,51 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+
+.. include:: ../../common.defs
+
+.. _admin-plugins-access_control:
+
+
+Traffic Dump Plugin
+*******************
+
+Description
+===========
+
+``Traffic Dump`` captures session traffic and writes to a replay file for each captured session. It then can be used to replay traffic for testing purpose.
+
+Plugin Configuration
+====================
+
+* ``Traffic Dump`` is a global plugin and is configured via :file:`plugin.config`.
```

Comment:
Same as other plugin documtation, try to use the "program" and "options" support in Sphinx.